File: benchmark/results/java-multilang/run-001/b-baseline/output/com/metaengine/demo/inventory/value_objects/WarehouseLocation.java
package com.metaengine.demo.inventory.value_objects;

import java.time.Instant;

/** WarehouseLocation value object for the inventory domain. */
public record WarehouseLocation(
    String id,
    Instant createdAt,
    Instant updatedAt,
    String name,
    String description
) {}

File: benchmark/results/python-multilang/run-005/b-baseline/output/metaengine_demo/inventory/value_objects/warehouse_location.py
"""WarehouseLocation value object for the inventory domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class WarehouseLocation:
    """WarehouseLocation value object."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
